JS/models/ExperimentalTypes.js

```diff
@@ -1,8 +1,8 @@
-const ExperimentalTypes   = {
-    LIFTING_BODY: 'lifting_body',
-    HYPERSONIC: 'HYPERSONIC',
-    HIGH_ALTITUDE: 'HIGH_ALTITUDE',
-    VTOL: "VTOL"
+const experimentalTypes   = {
```

Comment:
Usually, we use CAPITAL_LETTERS notation for constants. Please correct.
Also, make a common convention for key values. For example: `Lifting body`, `hypersonic`; or `LIFTING BODY`, `HYPERSONIC`, etc. (as you like).

Reply:
Done